readme.md

```diff
@@ -1,6 +1,6 @@
-# MapboxSpeech
+# Mapbox Speech
 
-A module for using the Mapbox Speech API specifically tuned for the [Mapbox Navigation SDK](https://www.mapbox.com/navigation/).
+Mapbox Speech connects your iOS application to the Mapbox Voice API. Take turn instructions from the [Mapbox Directions API](https://www.mapbox.com/api-documentation/#directions) and read them aloud naturally in multiple languages. This library is specifically designed to work with [MapboxDirections.swift](https://github.com/mapbox/MapboxDirections.swift/) as part of the [Mapbox Navigation SDK for iOS](https://github.com/mapbox/mapbox-navigation-ios/).
```

Comment:
> It might make sense to say that this should only be used in the navigation sdk.

This document points out that the developer would automatically use this library by using the navigation SDK and also suggests that a developer using MapboxDirections.swift directly could pair it with this library. Is the latter problematic? One could argue that MapboxDirections.swift should only really be used with the navigation SDK, but we haven’t gone that far yet.

Reply:
I like this cross-referencing of how the components fit together.